### PR TITLE
Add panel blog

### PR DESCRIPTION
**Pull request by alileclerc**

### CODE REVIEW

File: website/blog/2023-08-17-scaling-presto-panel-blog.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Meta processes data at an exabyte scale, benefiting tens of thousands of users and enhancing their data warehouse infrastructure with Presto. 
+
+Uber focuses on delivering a seamless user experience and utilizes Presto to support various business use cases, including efficient data processing for Uber Eats deliveries and quick matching with rides. Presto plays a critical role in business operations at Presto. 
+
+Overall, these three companies showcase the reliability of Presto as a powerful SQL query engine for scaling data analytics. Whether it's exploring vast amounts of data, performing ETL tasks, or optimizing user experiences, Presto proves its effectiveness in meeting the diverse needs of organizations across different industries. As Presto continues to evolve and improve, it is expected to play an even more significant role in the data analytics landscape in the future. 
+
+You can see the full <a href="https://www.youtube.com/watch?v=KHd86oyd534&list=PLJVeO1NMmyqXm5_fuFoKyMfZWyT5jOeKh&index=14" style="color:blue;">panel presentation on the Presto Foundation Youtube channel</a>.
```
Comment (wanglinsong) [inline website/blog/2023-08-17-scaling-presto-panel-blog.md:66]: YouTube

Comment (alileclerc) [reply website/blog/2023-08-17-scaling-presto-panel-blog.md]: fixed

File: website/blog/2023-08-17-scaling-presto-panel-blog.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+This panel discussion highlighted the diverse use cases of scaling Presto for data analytics at Meta, Uber, and Intuit. Each company showcased their unique experiences and demonstrated how Presto has become a vital component of their data processing infrastructure. 
+
+![Panelists](/img/blog/2023-08-17-scaling-presto-panel-blog/Panelists.png)
+
+Intuit, with its massive data landscape serving millions of customers, relies on Presto on Spark for data exploration and operationalization. By leveraging Presto, they have achieved significant business impact, improved analyst productivity, and streamlined their tax season analytics. 
+
+Meta processes data at an exabyte scale, benefiting tens of thousands of users and enhancing their data warehouse infrastructure with Presto. 
+
+Uber focuses on delivering a seamless user experience and utilizes Presto to support various business use cases, including efficient data processing for Uber Eats deliveries and quick matching with rides. Presto plays a critical role in business operations at Presto.
```
Comment (wanglinsong) [inline website/blog/2023-08-17-scaling-presto-panel-blog.md:62]: Presto plays a critical role in business operations at **Uber**.

File: website/blog/2023-08-17-scaling-presto-panel-blog.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+### Future Plans 
+
+Meta plans to consolidate their data warehouse workloads onto Presto as they migrate from Spark SQL to Presto. By standardizing on Presto, Meta aims to simplify their data processing architecture and further optimize the performance and scalability of their analytical workflows. 
+
+## Presto at Uber - Enhancing User Experience with Presto  
+
+Gurmeet Singh, a software engineer in the Uber Presto team, discusses Uber's journey with Presto and its impact on user experience. Previously working in the storage domain, Gurmeet transitioned to the analytics layer at Uber, where he witnessed firsthand how Presto drives business use cases and improves user satisfaction. Key use cases are exploratory analytics for operations teams who want to do analysis on how operations are doing, as well as batch ETL workloads where you run a query on an ad hoc basis but you want to convert it to a pipeline. Dashboarding, reporting, and experimentation analysis is also done with Presto.
```
Comment (wanglinsong) [inline website/blog/2023-08-17-scaling-presto-panel-blog.md:46]: **are** also done

**Review by wanglinsong — APPROVED** [review]

**Pull request merged**